### DISCUSSION TITLE
Invoke HitTest by default

### DISCUSSION THREAD

**Discussion by RomanSoloweow** [AvaloniaUI/Avalonia#6976] [Q&A]
How to call HitTest by default? I have implemented inheritance from ICustomHitTest. 

And depending on the state of the element, I want to switch between mine and default HitTest

**Comment by RomanSoloweow** [comment]
@wieslawsoltes maybe you know :)

**Comment by maxkatz6** [comment]
Maybe this will help 
https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Visuals/Rendering/ICustomSimpleHitTest.cs#L28

**Reply by RomanSoloweow** [reply]
Unfortunately not. I tried. The behavior is different from the default
